File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    word = s.split(" ")
    string = ''
    
    for i in range(len(word)):
        string = word[i]
        for j in range(len(string)):
            if j%2==0:
                answer += string[j].upper()
            else:
                answer += string[j].lower()
        if i!=len(word)-1:
            answer += ' '
    
    return answer

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    
    var arr = s.split(" ");
    for(let a of arr){
        for(let i=0; i<a.length; i++){
            if(i%2===0) answer += a[i].toUpperCase();
            else answer += a[i].toLowerCase();
        }
        answer += ' '
    }
    
    return answer.substr(0, answer.length-1);
}